### PR TITLE
Unify UI screen titles and minor lang. fixes

### PR DESCRIPTION
**Pull request by rkratky**
* Unifies installer screen titles to use '&lt;Topic&gt; configuration' (where approapriate).
* Fixes minor lang. problems and typos (commas, Ubuntu Po -> Pro, Internet -> internet, ...).

(Fixes [FR-5900](https://warthogs.atlassian.net/browse/FR-5900))

[FR-5900]: https://warthogs.atlassian.net/browse/FR-5900?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: subiquity/ui/views/ubuntu_pro.py [hunk 1]
```diff
@@ -832,7 +832,7 @@ def itemize(item: str, marker: str = "•") -> Columns:
             Text(""),
             Pile([itemize(svc, marker="  •") for svc in services]),
             Text(""),
-            Text(_("Ubuntu Pro is free for personal use on up to 3 machines.")),
+            Text(_("Ubuntu Pro is free for personal use on up to three machines.")),
```
Comment (ogayot) [inline subiquity/ui/views/ubuntu_pro.py:835]: @lechsandecki do you know if we have other occurrences of this text outside of the server installer?

**Review by ogayot — COMMENTED** [review]
Thanks for the updates!

File: subiquity/ui/views/ubuntu_pro.py [hunk 2]
```diff
@@ -221,8 +221,8 @@ class UpgradeYesNoFormNoNetwork(UpgradeYesNoForm):
         _("Skip Ubuntu Pro setup for now"),
         help="\n"
         + _(
-            "Once you are connected to the Internet, you can"
-            " enable Ubuntu Po via the 'pro attach' command."
```
[the remark this answers is not in the document]
Comment (rkratky) [reply subiquity/ui/views/ubuntu_pro.py:225]: Done: https://github.com/canonical/subiquity/pull/1876/commits/63a0d8d7ae0de9ff47aecded189b38d734b4d010

File: subiquity/ui/views/ubuntu_pro.py (hunk 1 above)
Comment (Chris-Peterson444) [reply subiquity/ui/views/ubuntu_pro.py:835]: According to this it's five machines now https://ubuntu.com/pro/subscribe

File: subiquity/ui/views/ubuntu_pro.py [hunk 3]
```diff
@@ -221,8 +221,8 @@ class UpgradeYesNoFormNoNetwork(UpgradeYesNoForm):
...
+            "Once you are connected to the internet, you can"
+            " enable Ubuntu Pro using the 'pro attach' command."
```
Comment (Chris-Peterson444) [inline subiquity/ui/views/ubuntu_pro.py:225]: Could you reflect these changes in the class docstring as well?

File: subiquity/ui/views/ubuntu_pro.py (hunk 1 above)
Comment (Chris-Peterson444) [inline subiquity/ui/views/ubuntu_pro.py:835]: Same as above

**Review by Chris-Peterson444 — APPROVED** [review]
Looks good to me! Just a small request.

File: subiquity/ui/views/ubuntu_pro.py (hunk 3 above)
Comment (rkratky) [reply subiquity/ui/views/ubuntu_pro.py:225]: Of course. Comment blindness :)

File: subiquity/ui/views/ubuntu_pro.py (hunk 1 above)
Comment (rkratky) [reply subiquity/ui/views/ubuntu_pro.py]: Updated to five.

**Review by Chris-Peterson444 — APPROVED** [review]

**Pull request merged**